quickstarts/dbmarlin/config.yml

```diff
@@ -0,0 +1,61 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: dbmarlin
+
+# Displayed in the UI (required)
+title: DBmarlin
+
+# Long-form description of the quickstart (required)
+description: |
+  The DBmarlin quickstart allows you to get visibility into the performance and availability of your database services from within New Relic.
+
+  The installation instructions for the quickstart are available at https://newrelic.com/instant-availability/dbmarlin and https://docs.dbmarlin.com/docs/integrations/newrelic.
```

Comment:
```suggestion

```

This line is unnecessary, we'll want to add the integration doc under data sources.